### PR TITLE
528 update search page

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#528

---

#### Any info the reviewer needs to know to test 
just  navigate to `/hosts/` route and check it
---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/components/Pages/SearchHosts/Form.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    </T.H3C>
+    <T.PXL mt="1" mb="6">
+      You can search for hosts by filling in the city and dates you&apos;re
+      looking for. Need somewhere quickly? Then select automatic bookings.
+    </T.PXL>
+    <Row>
+      <T.PL mb="1" mr="4">
+        I want a place to stay in
+      </T.PL>
+      <Select
```
Comment (thejoefriel) [inline client/src/components/Pages/SearchHosts/Form.js:39]: Can we remove the autofill option as when I click on the text my device tries to give me a suggestion like "LONDON" and I can't actually click the button in the dropdown

File: client/src/components/Pages/SearchHosts/NoResults.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <T.H4C mt="8" mb="5">
+      {within7Days
+        ? 'Sorry, it is too close to your requested date to stay'
+        : 'Sorry, No hosts are available'}
+    </T.H4C>
+    {within7Days ? (
+      <T.PXL>
+        If you do not have anywhere to stay for your upcoming internship and it
+        is urgent, please email PressPad at{' '}
+        <T.Link to="urgent@presspad.co.uk" color="lightBlue">
```
Comment (thejoefriel) [inline client/src/components/Pages/SearchHosts/NoResults.js:16]: this isn't showing pointer on it - maybe need to add ```cursor: pointer``` in the css

File: client/src/components/Common/Typography/index.js [hunk 3]
```diff
@@ -81,15 +81,15 @@ const PXSBold = ({ children, ...props }) => (
...
 );
 
-const Link = ({ children, to, isExternal, ...props }) => (
+const Link = ({ children, to, isExternal, caps, ...props }) => (
   <StyledLink
     to={isExternal ? undefined : to}
     href={!isExternal ? undefined : to}
     as={!isExternal ? undefined : 'a'}
     rel={!isExternal ? undefined : 'noopener noreferrer'}
     target={!isExternal ? undefined : '_blank'}
```
Comment (thejoefriel) [inline client/src/components/Common/Typography/index.js:90]: I think you can just change these to ```target={!isExternal && '_blank'}``` and then the undefined would be automatically set if it didn't pass this condition right? Would that be cleaner?

File: client/src/components/Pages/SearchHosts/NoResults.js (hunk 2 above)
Comment (thejoefriel) [reply client/src/components/Pages/SearchHosts/NoResults.js:16]: Also the link text size is noticeably smaller than the main body text. Can we make these the same size please?

Comment (RamyAlshurafa) [reply client/src/components/Pages/SearchHosts/NoResults.js:16]: you mean bigger right?

File: client/src/components/Common/Typography/index.js (hunk 3 above)
Comment (RamyAlshurafa) [reply client/src/components/Common/Typography/index.js:90]: `target={!isExternal && '_blank'}` return true or false, and that will throw a warning message in the console

```
Warning: Failed prop type: Invalid prop `target` of type `boolean` supplied to `Link`, expected `string`.
    in Link (created by Styled(Link))
...
```

Comment (thejoefriel) [reply client/src/components/Common/Typography/index.js:90]: Ah good to know!

File: client/src/components/Pages/SearchHosts/NoResults.js (hunk 2 above)
Comment (thejoefriel) [reply client/src/components/Pages/SearchHosts/NoResults.js:16]: yeah, it's a bit small at the moment

Comment (thejoefriel) [reply client/src/components/Pages/SearchHosts/NoResults.js:16]: You can see here: 

image

**Comment by RamyAlshurafa** [comment]
@thejoefriel you mean in the date pickers? the date pickers just disallow user from selecting dates in the past, or select a start date later than the end date, and they don't get any dates from the backend.

File: client/src/components/Pages/SearchHosts/NoResults.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/components/Pages/SearchHosts/NoResults.js:16]: @thejoefriel 
the link font was bigger than the body text, then i let both of them same font-size =14px, and i have no idea why they aren't don't look the same

**Comment by thejoefriel** [comment]
> @thejoefriel you mean in the date pickers? the date pickers just disallow user from selecting dates in the past, or select a start date later than the end date, and they don't get any dates from the backend.

That's what I mean though. I selected a start date in the search for April 2025, but the listing still showed up even though in the listing doesn't have any available dates beyond 2020?

File: client/src/components/Pages/SearchHosts/NoResults.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/components/Pages/SearchHosts/NoResults.js:16]: just find out why it wasn't working

**Pull request merged**